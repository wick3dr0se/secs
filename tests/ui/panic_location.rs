//@ run: 101

use secs::World;

fn main() {
    let mut world = World::default();

    let id = world.spawn((1_u32,));
    world.spawn((10_u32, "foo"));
    world.despawn(id);

    world.get::<u32>(id);
}
