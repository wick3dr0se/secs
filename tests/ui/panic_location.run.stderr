
thread 'main' panicked at tests/ui/panic_location.rs:12:11:
Getting `u32` from despawned entity (despawned at tests/ui/panic_location.rs:10:11)
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
